client/src/components/QandA/QuestionListEntry.jsx

```diff
@@ -1,16 +1,19 @@
 /* eslint-disable react/prop-types */
 import React from 'react';
 import Moment from 'moment';
+import AddAnswer from './AddAnswer.jsx';
 
-const QuestionListEntry = ({ data, answers }) => {
+const QuestionListEntry = ({ data, answers, currentProductName }) => {
   return (
     <div className="container">
       <div className="row">
         <div className="col-9">
           <h5>Q: {data.question_body}</h5>
         </div>
         <div className="col-3">
-          <p className="text-end">Helpful? Yes ({data.question_helpfulness}) | Add Answer</p>
+          <div className="row">
+            <p className="text-end">Helpful? Yes ({data.question_helpfulness}) | <AddAnswer currentProductName={currentProductName} currentQuestionBody={data.question_body} /></p>
```

Comment:
Should clicking on the 'Yes' trigger a PUT request? if so, feel free to reference my ReviewListEntry.jsx

Reply:
It should and I have been bad about leaving comments clarifying where and when I plan to implement things. Thanks for pointing this out and I will take a look-see :)